build.sbt

```diff
@@ -52,36 +34,29 @@ val all_scala_versions = Seq(scala212, scala213)
 val default_scala_version = settingKey[String]("Default Scala version")
 Global / default_scala_version := scala212
 
-val LATEST_RELEASED_SPARK_VERSION = "3.5.3"
+val LATEST_RELEASED_SPARK_VERSION = "3.5.0"
 val SPARK_MASTER_VERSION = "4.0.0-SNAPSHOT"
 val sparkVersion = settingKey[String]("Spark version")
 spark / sparkVersion := getSparkVersion()
-connectCommon / sparkVersion := getSparkVersion()
-connectClient / sparkVersion := getSparkVersion()
-connectServer / sparkVersion := getSparkVersion()
-sharing / sparkVersion := getSparkVersion()
 
 // Dependent library versions
 val defaultSparkVersion = LATEST_RELEASED_SPARK_VERSION
 val flinkVersion = "1.16.1"
-val hadoopVersion = "3.3.4"
+val hadoopVersion = "3.3.6"
 val scalaTestVersion = "3.2.15"
 val scalaTestVersionForConnectors = "3.0.8"
 val parquet4sVersion = "1.9.4"
 
 // Versions for Hive 3
```

Comment:
seems like we rather should add Hive 4 here, as there is already 2 and 3 instead of changing the versions for Hive 3?

Reply:
+1. May be create a separate target for Hive 4, just like we have for hive 2 and hive 3.